app.py

```diff
@@ -64,7 +64,7 @@ def verify_password(user_id, password):
 
 @app.route('/forms', methods=['POST'])
 @auth.login_required
-@cross_origin(origins=FRONTEND_URL)
+@cross_origin(origins='*')
```

Comment:
Why?

Reply:
The test fails otherwise. I'll try to put an array with the front-end URL and localhost.

Reply:
Strange because manual tests with local instances worked.